src/components/About.js

```diff
@@ -3,11 +3,13 @@ import { Icon } from "@iconify/react";
 import angularIcon from "@iconify/icons-logos/angular-icon";
 import reactIcon from "@iconify/icons-logos/react";
 import vueIcon from "@iconify/icons-logos/vue";
+import myProfileImage from "./assets/myProfile.jpg"; // Đường dẫn có thể khác tùy theo cấu trúc thư mục của bạn
```

Comment:
Let's avoid using Vietnamese in code even for comments. I'd use either English or remove this comment if it's copied from somewhere